src/framework/components/particlesystem/particlesystem_system.js

```diff
@@ -197,10 +197,35 @@ pc.extend(pc, function() {
             }, {
                 name: "spawnBounds",
                 displayName: "Spawn Bounds",
-                description: "Defines an AABB in which particles are allowed to spawn",
+                description: "Defines an AABB/radius in which particles are allowed to spawn",
```

Comment:
I agree that we should have different properties per shape and show / hide them accordingly in the UI, like we do for collision components. 

As for names I'd say emitterExtents or emitterHalfExtents and emitterRadius for spheres.